build.sbt

```diff
@@ -1,10 +1,14 @@
+import scala.util.Try
+
 val scala3Version   = "3.4.2"
 val scala212Version = "2.12.19"
 val scala213Version = "2.13.14"
 
 val zioVersion       = "2.1.1"
 val scalaTestVersion = "3.2.18"
 
+val isCI = Try(sys.env.getOrElse("CI", "false").toBoolean).getOrElse(false)
```

Comment:
Could this flat be specific to the workaround instead of `CI`? Should the workaround be disabled in CI?